airbase/airtable.py

```diff
@@ -131,8 +131,15 @@ async def get_bases(self) -> Optional[List]:  # noqa: F821
 
     async def get_base(self, value: str, key: Optional[str] = None):
         assert key in (None, "id", "name")
+
+        if key == "id":
+            return Base(
+                base_id=value, session=self._session, logging_level="info",
+            )
+
         if not getattr(self, "bases", None):
             await self.get_bases()
+
         if self.bases:
```

Comment:
`get_base` should first check for attr `bases` in self to allow search by `id` in `self._bases_by_id` which will return a `Base` class with further attributes